docs/disco.sh

```diff
@@ -191,6 +196,28 @@ create-nfs () {
     touch /nfs-pool/.witness.txt
 }
 
+extend-nfs () {
+    source /coclo/configuration.env
+
+    local size=$1
+    local project=$(get-project-name)
+    local new_disk_id=$NFS_SERVER_NAME-$RANDOM
+
+    # Create an additional disk
+    gcloud compute disks create $new_disk_id \
+            --zone $GCLOUD_ZONE --project $project \
+            --size $size --type "pd-standard"
+
+    # # Attach it pseudo-physically to the main NFS VM
+    gcloud compute instances attach-disk $NFS_SERVER_NAME \
```

Comment:
not necessary, but if you could, would be very nice to allow $2 to optionally set the NFS_SERVER_NAME (so this can be used more generally).